Procfile

```
@@ -1 +1 @@
-web: cd apps && panel serve --address="0.0.0.0" --port=$PORT *.py --allow-websocket-origin=tec-hatch-demo.herokuapp.com --allow-websocket-origin=dev-tec-hatch-demo.herokuapp.com
+web: panel serve --address="0.0.0.0" --port=$PORT apps/hatch.py --allow-websocket-origin=tec-hatch-demo.herokuapp.com --allow-websocket-origin=dev-tec-hatch-demo.herokuapp.com
```

```suggestion
web: panel serve --address="0.0.0.0" --port=$PORT apps/*.py --allow-websocket-origin=tec-hatch-demo.herokuapp.com --allow-websocket-origin=dev-tec-hatch-demo.herokuapp.com
```

We want to run both `hatch` and `test-hatch` dashboards.